.gitignore

```diff
@@ -17,3 +17,18 @@
 #output files
 /build/
 /linux/installer/bin/*.bin
+
+# files downloaded in preparation phase
+Intel redistributable binary.txt
+Master_EULA_for_Intel_Sw_Development_Products.pdf
+external/ippcp_internal/inc/ippcp.h
```

Comment:
Perhaps I am misunderstanding something, but as far as I understand this would ignore the entire content under [external/ippcp_internal](https://github.com/intel/linux-sgx/tree/master/external/ippcp_internal) which contains the `ipp-crypto` submodule, the `Makefile`, patches, etc.

Reply:
You are right. No need to change this.